### DISCUSSION TITLE
Is there a way to force a specific dependency check despite prConcurrentLimit is reached?

### DISCUSSION THREAD

**Discussion by stavrogin** [renovatebot/renovate#16437] [Request Help] [answered]
Hi!

I saw that Renovate will create PRs for security update dependencies even if prConcurrentLimit is reached
https://docs.renovatebot.com/configuration-options/#prconcurrentlimit

My question is: is it possible to force also some specific dependencies to be checked despite prConcurrentLimit is reached?

For example: prConcurrentLimit is set to 4 and Renovate already created 4 PRs, but I would like to have always a PR created if there is any update on -let's say- jackson java library. 

Is there any way to do that?

**Comment by rarkins** [comment, marked as answer]
Not that I can think of. You will need to force it using the Dependency Dashboard checkbox
